src/DbInterface.cpp

```diff
@@ -864,6 +864,46 @@ void DbInterface::getSoCIpAddress(std::shared_ptr<swss::DBConnector> configDbCon
     processSoCIpAddress(entries);
 }
 
+//
+// ---> getMuxModeConfig();
+//
+// retrieve MUX mode configuration
+//
+std::map<std::string, std::string> DbInterface::getMuxModeConfig()
+{
+    MUXLOGINFO("Reading MUX mode configuration");
+    std::shared_ptr<swss::DBConnector> configDbPtr = std::make_shared<swss::DBConnector> ("CONFIG_DB", 0);
+    swss::Table configDbMuxCableTable(configDbPtr.get(), CFG_MUX_CABLE_TABLE_NAME);
```

Comment:
Can `configDbConnector` be used here?

Reply:
> Can `configDbConnector` be used here?

Conventionally I have always used `DBConnector`, is there any concern with it? Or does `configDbConnector` has more benefits?